src/views/textview/text-view.js

```diff
@@ -79,21 +78,7 @@ export class TextView extends LitElement {
   }
 
   handleFolioChanged() {
-    let segment = '';
-    if (this.lang === LANGUAGE_CODES.CHINESE) {
-      const folio = parseInt(this.folio)
-        .toString()
-        .padStart(3, '0');
-      segment = `${this.fileName}:${folio}-1`;
-    } else if (this.lang === LANGUAGE_CODES.TIBETAN) {
-      segment = `${this.fileName}:${this.folio}-0`;
-    } else {
-      if (this.fileName.startsWith('dhp')) {
```

Comment:
This will no longer work for Pali ..
So I guess this now has to be done in the backend.

Reply:
Indeed, we shouldn't do that on the frontend anymore.